code/game/atoms.dm

```diff
@@ -36,7 +36,7 @@ var/global/list/ghdel_profiling = list()
 	//Detective Work, used for the duplicate data points kept in the scanners
 	var/list/original_atom
 
-	var/list/beams
+	var/list/beams = list()

```

Comment:
Its done as necessary, just like materials datums, do you see every instance in the game having a materials datum as well?